Library/Formula/unbound.rb

```diff
@@ -19,13 +19,10 @@ def install
                           "--sysconfdir=#{etc}",
                           "--with-libevent=#{Formula["libevent"].opt_prefix}",
                           "--with-ssl=#{Formula["openssl"].opt_prefix}"
+    inreplace "doc/example.conf", 'username: "unbound"', "username: \"#{ENV["USER"]}\""
```

Comment:
Or could just make the `inreplace` essentially unconditional: `inreplace etc/"unbound/unbound.conf", 'username: ".*"', "username: \"#{ENV["USER"]}\""`